src/EnergyPlus/UnitarySystem.cc

```diff
@@ -14094,30 +14138,93 @@ namespace UnitarySystems {
+                if (this->m_LastMode == HeatingMode) {
+                    this->m_HeatingAuxElecConsumption += this->m_AncillaryOffPower * (1.0 - this->m_PartLoadFrac) * ReportingConstant;
+                }
+
+                ElecHeatingPower = DataHVACGlobals::DXElecHeatingPower;
+
+            } else if ((SELECT_CASE_var == DataHVACGlobals::Coil_UserDefined) || (SELECT_CASE_var == DataHVACGlobals::Coil_HeatingWater) ||
+                       (SELECT_CASE_var == DataHVACGlobals::Coil_HeatingSteam) || (SELECT_CASE_var == DataHVACGlobals::Coil_HeatingDesuperheater)) {
+
+                if (HeatingLoad) {
+                    this->m_TotalAuxElecPower =
+                        this->m_AncillaryOnPower * this->m_PartLoadFrac + this->m_AncillaryOffPower * (1.0 - this->m_PartLoadFrac);
+                    this->m_HeatingAuxElecConsumption = this->m_AncillaryOnPower * this->m_PartLoadFrac * ReportingConstant;
+                }
+                if (this->m_LastMode == HeatingMode) {
+                    this->m_HeatingAuxElecConsumption += this->m_AncillaryOffPower * (1.0 - this->m_PartLoadFrac) * ReportingConstant;
+                }
+
+            } else if (this->m_HeatCoilExists) {
+                if (HeatingLoad) {
+                    // if discrete, the coil cycles on and off
+                    this->m_TotalAuxElecPower =
+                        this->m_AncillaryOnPower * this->m_PartLoadFrac + this->m_AncillaryOffPower * (1.0 - this->m_PartLoadFrac);
+                    this->m_HeatingAuxElecConsumption = this->m_AncillaryOnPower * this->m_PartLoadFrac * ReportingConstant;
+                }
+                if (this->m_LastMode == HeatingMode) {
+                    this->m_HeatingAuxElecConsumption += this->m_AncillaryOffPower * (1.0 - this->m_PartLoadFrac) * ReportingConstant;
+                }
+                ElecHeatingPower = DataHVACGlobals::ElecHeatingCoilPower;
+            }
+        }
+
+        if (!HeatingLoad && !CoolingLoad) {
+            if (this->m_TotalAuxElecPower > 0.0) {
+                this->m_TotalAuxElecPower = this->m_TotalAuxElecPower;
             }
+            this->m_TotalAuxElecPower = this->m_AncillaryOffPower;
         }
 
+        if (this->m_SuppCoilExists) {
+            auto const SELECT_CASE_var(this->m_HeatingCoilType_Num);
+            if (SELECT_CASE_var == DataHVACGlobals::Coil_HeatingElectric) {
+                // Yuck, what if main heating coil and supp heating coil are electric
+                // Probably need to save heating coil power when they are simulated to split out main and supp heater
+                SuppHeatingPower = DataHVACGlobals::ElecHeatingCoilPower;
+            }
+        }
+
+        this->m_ElecPower = locFanElecPower + ElecCoolingPower + ElecHeatingPower + SuppHeatingPower + DataHVACGlobals::ElecHeatingCoilPower +
+                            this->m_TotalAuxElecPower;
+        this->m_ElecPowerConsumption = this->m_ElecPower * ReportingConstant;
```

Comment:
Need to remove DataHVACGlobals::ElecHeatingCoilPower in this->m_ElecPower equation.